CRM/Financeexportformats/Hook/AlterContent/FinancialBatchPage.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+class CRM_Financeexportformats_Hook_AlterContent_FinancialBatchPage {
+ 
+  public function __construct() {
+  }
+
+  /**
+   * @param string $tplName
+   * @param string $content
+   */
+  public function run($tplName, &$content)
+  {
+    if ($this->shouldRun($tplName)) {
```

Comment:
I find code with less nesting more readable in general, so you can probably do:

```
if (!$this->shouldRun()) {
  return;
}

// rest of the code
```

also probably better to move this into separate method with a name that indicates what it does, making things a bit more readable:

```
private $pageContent;

public function __construct(&$content) {
  $this->pageContent = &$content;
}

public function run() {
  if (!$this->shouldRun()) {
    return;
  }

  this->addInlineQuickbooksExportSelectOption();
}

private function addInlineQuickbooksExportSelectOption() {
  $this->pageContent = str_replace(
    "<option value=\"CSV\">CSV</option>",
    "<option value=\"CSV\">CSV</option><option value=\"QuickbooksCSV\">Quickbooks CSV</option>",
    $this->pageContent
  );
}
```